docker/include/bin/start_faux

```diff
@@ -68,6 +68,13 @@ elif [ -n "${options[passwordfail]}" ]; then
     service ssh start
 fi
 
+# security.nmap.http faux device setup
+if [ -n "${options[http]}" ]; then
+    cp /root/nginx/nginxpass.conf /etc/nginx/nginx.conf
+    sed -i 's/listen 80;/listen 12345; listen 54321;/' /etc/nginx/nginx.conf
```

Comment:
What is this doing?  If I read it correctly, it's replacing port 80 with ports 12345 and 54321?  Given the 'http' option, I would expect this to be listening on port 80... ?

Reply:
This test checks if http servers are open on arbitrary ports, so the faux test does just this for checking that the code works.

Reply:
My question then is if this argument should be "http" which I would normally expect to be "run http server on port 80", or something else like ohttp (other http) that means "run an http server on other ports"?  This isn't about "what the test checks", as such -- rather, "what behavior does the faux device have?"  The faux should be coherent independent of the test :-).

Reply:
Sure, I can see your perspective now, "ohttp" sounds good, I'll amend it.